src/components/file-uploader/file-uploader.hbs

```diff
@@ -11,7 +11,7 @@
   <p class="{{@root.prefix}}--label-description">only .jpg and .png files. 500kb max file size.</p>
   <div class="{{@root.prefix}}--file" data-file>
     <label for="your-file-importer-id-here"
-      class="{{@root.prefix}}--file-btn {{@root.prefix}}--btn {{@root.prefix}}--btn--primary"
+      class="{{@root.prefix}}--file-btn {{@root.prefix}}--btn {{@root.prefix}}--btn--primary {{#if componentsX}}{{else}}{{@root.prefix}}--btn--sm{{/if}}"
```

Comment:
@asudoh This feels like it might be a little sketchy -- let me know if there's a smarter way to do this. 👍

Reply:
you may want to use `unless` here instead

Reply:
Ah, exactly! Thanks 💪